hawc/main/celery.py

```diff
@@ -24,32 +24,39 @@ def debug_task(self):
     "worker-healthcheck": {
         "task": "hawc.apps.common.tasks.worker_healthcheck",
         "schedule": timedelta(minutes=5),
+        "options": {"expires": timedelta(minutes=5)},
```

Comment:
does expires accept a timedelta?  I'd assume it does, just didn't see it in the docs?

Reply:
I would assume it does too, but I'll double check and comment it here.

Reply:
The [options are the same as in `apply_async`](https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#available-fields), which has an [example of `expires` using `timedelta`](https://docs.celeryproject.org/en/stable/userguide/calling.html#expiration).

Reply:
I stand corrected, it looks like it may only accept a float/datetime [[1](https://github.com/celery/celery/blob/9e435228cb106588f408ae71b9d703ff81a80531/celery/app/task.py#L441),[2](https://github.com/celery/celery/blob/9e435228cb106588f408ae71b9d703ff81a80531/celery/app/amqp.py#L301)]. I will call `total_seconds()` on each of these `timedelta`s to convert it to seconds just to be safe.